apps/andi/lib/andi/input_schemas/datasets/dataset.ex

```diff
@@ -77,18 +107,33 @@ defmodule Andi.InputSchemas.Datasets.Dataset do
   def validate_unique_system_name(%{changes: %{technical: technical}} = changeset) do
     id = Ecto.Changeset.get_field(changeset, :id)
     data_name = Ecto.Changeset.get_change(technical, :dataName)
-    org_name = Ecto.Changeset.get_change(technical, :orgName)
+    org_id = Ecto.Changeset.get_field(changeset, :organization_id)
+
+    case is_nil(org_id) do
```

Comment:
Consider breaking this case into a function and passing the result to one copy of the duplicated lines below

Reply:
Fixed!